test/focus.html

```diff
@@ -104,13 +105,50 @@
       });
     });
 
+    test('list focus change with down arrow', function(done) {
+      container.useTabIndex = true;
+      list.items = buildDataSet(100);
+
+      flush(function() {
+        var initialItem = getFirstItemFromList(list);
+        var itemToFocus = getNthItemFromList(list, 1);
+        initialItem.addEventListener('focus', function(e) {
+          MockInteractions.pressAndReleaseKeyOn(list, 40); // down
+        });
+        itemToFocus.addEventListener('focus', function(e) {
+          var focusedIndex = list.modelForElement(e.target).index;
+          assert.equal(focusedIndex, 1);
+          done();
+        });
+        MockInteractions.focus(initialItem);
+      });
+    });
+
+    test('list focus change with up arrow', function(done) {
+      list.items = buildDataSet(100);
+
+      flush(function() {
+        var initialItem = getNthItemFromList(list, 1);
+        var itemToFocus = getFirstItemFromList(list);
+        initialItem.addEventListener('focus', function(e) {
+          MockInteractions.pressAndReleaseKeyOn(list, 38); // up
+        });
+        itemToFocus.addEventListener('focus', function(e) {
+          var focusedIndex = list.modelForElement(e.target).index;
+          assert.equal(focusedIndex, 0);
+          done();
+        });
+        MockInteractions.focus(initialItem);
+      });
+    });
```

Comment:
@keanulee, updated the focus tests to rely on `flush()` and `element.tabIndex` as in some of the other tests in `focus.html`, so I'm finally getting greens on the remote testing pass!!